client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java

```diff
@@ -1046,7 +1131,7 @@ private void indexSearchTestData() throws IOException {
         assertTrue(authorsResponse.isAcknowledged());
 
         CreateIndexRequest reviewersRequest = new CreateIndexRequest("contributors")
-            .mapping("doc", "user", "type=keyword");
+            .mapping("doc", "user", "type=keyword,store=true");
```

Comment:
was this change necessary?

Reply:
I have tested the ["stored_fields" flag](https://github.com/elastic/elasticsearch/pull/31387/files#diff-da6eb90ef1049f45aefed2e1dc3d715cR866) in `testExplain()` and I add an assertion [here](https://github.com/elastic/elasticsearch/pull/31387/files#diff-da6eb90ef1049f45aefed2e1dc3d715cR897), if we don't "store" this field we will not get any values in `GetResult`'s `fields`, so this assertion will not work. Otherwise we remove this assertion ?

Reply:
I have tested the ["stored_fields" flag](https://github.com/elastic/elasticsearch/pull/31387/files#diff-da6eb90ef1049f45aefed2e1dc3d715cR866) in `testExplain()` and I add an assertion [here](https://github.com/elastic/elasticsearch/pull/31387/files#diff-da6eb90ef1049f45aefed2e1dc3d715cR897), if we don't "store" this field we will not get any values in `GetResult`'s `fields`, so this assertion will not work. Otherwise we remove this assertion ?

Reply:
got it, it's fine, I got confused by the github diff, it seemed like this was part of testFieldCaps only, but it's shared between different methods. All good!